sktime/utils/_testing/scenarios_classification.py

```diff
@@ -117,6 +121,31 @@ def args(self):
     default_arg_sequence = ["fit", "predict", "predict", "predict"]
 
 
+class ClassifierFitPredictThreeClasses(ClassifierTestScenario):
+    """Fit/predict with univariate panel X, pd-multiindex mtype, and three classes."""
+
+    _tags = {
+        "X_univariate": True,
+        "X_unequal_length": False,
+        "is_enabled": True,
+        "n_classes": 3,
```

Comment:
Would there be any harm in going to more than three classes? I understand that we are interested in multi-class prediction rather than three-class prediction, so e.g., five classes might catch more errors. Or perhaps this test can be made parametric with anywhere from three to e.g., 15 classes to increase coverage further (at the cost of more runtime)?

Reply:
Yes, you are right, of course - it is runtime that worries me as you say, and the minimum sample size requirement.

Some classifiers have grid search internally, 5-fold as default. So you'd want to see at least 3 instances of each in the training set, which gets you to `n_instances = n_classes * 4` (or better 5, 6).

Many classifiers have between second and third power scaling on the number of instances, and there are a large number of classifiers.

We can of course check how much it is, if you would like to, I do not mind - though I wonder if 5 classes gives that much more coverage than 3. My gut feeling is, if sth breaks with 5, it already breaks with 3, e.g., the one-hot encoder example.